src/buildCommon.js

```diff
@@ -238,7 +238,8 @@ const makeOrd = function<NODETYPE: "spacing" | "mathord" | "textord">(
     if (value.charCodeAt(0) === 0xD835) {
         // surrogate pairs get special treatment
         const [wideFontName, wideFontClass] = wideCharacterFont(value, mode);
-        return makeSymbol(value, wideFontName, mode, options, [wideFontClass]);
+        return makeSymbol(value, wideFontName, mode, options,
+			["mord", wideFontClass]);
```

Comment:
How about `classes.concat(wideFontClass)`?

Reply:
@ylemkimon Yes, that will work. Is that faster?

Reply:
The benefit is that if someone adds something to `classes` they don't have to also update this line.

Reply:
Ah, that makes sense. Thank you for pointing it out.